orchestra/migrations/0079_step_todolist_templates_to_apply.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
to clean up the migration history, you can revert the two migrations here and create one migration that creates the foreign key-based model.